cmd/client/commands.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func reload_process() {
+	fmt.Println("Really restart the remote supervisord process y/N?")
+	buf := bufio.NewReader(os.Stdin)
+	response, err := buf.ReadByte()
+	switch {
+	case err != nil:
+		os.Exit(1)
```

Comment:
do not exit on failure (or maybe do, depends on supervisor behaviour) -> try with `<C-d>`